arcade/text.py

```diff
@@ -234,7 +236,7 @@ def group(self, group: pyglet.graphics.Group):
     @property
     def value(self) -> str:
         """
-        Get or set the current text string to display.
+        Current text string to display.
```

Comment:
It's important to communicate if the property is read/write. That's the reason for the "get or set" part.

This applies to most or all propreties here.

Reply:
It's kind of repetitive, though

Reply:
If it's read only, it's usually stated that `Read-only` to clarify.

Reply:
It's definitely repetative, but it's explicit not creating doubt. We've had so many examples of users not understanding the properties can actually be set. It's immediately visible in your IDE, python `help()` and in sphinx.